### DISCUSSION TITLE
Nice to have options that take care of auto-merge

### DISCUSSION THREAD

**Discussion by Enigama** [pwntester/octo.nvim#622] [Ideas]
![image](https://github.com/user-attachments/assets/04f7b6f3-9cb8-42a8-b7c0-a6dcd56d21a5)

**Comment by williambdean** [comment]
Do you know the gh CLI command for this?

**Reply by Enigama** [reply]
@williambdean hm I'm not sure if it helps but here what I found
https://cli.github.com/manual/gh_pr_merge
![image](https://github.com/user-attachments/assets/00b956e3-9bd6-4a30-be7f-159ae19240cb)

**Reply by Enigama** [reply]
damn, it's for merging

**Reply by Enigama** [reply]
So currently it is not supported by CLI

**Reply by williambdean** [reply]
We can use graphql if needed. It would be a mutation

**Reply by Enigama** [reply]
> We can use graphql if needed. It would be a mutation

I've created a feature request to gh cli team maybe it will be added

**Reply by Enigama** [reply]
So I reached out to gh cli team and got some info how to achive this:

https://github.com/cli/cli/issues/10651

https://github.com/cli/cli/issues/10651#issuecomment-2748214415

@williambdean is it possible to do in octo?

**Reply by williambdean** [reply]
Yeah, seems easy to do. You can already add this to a custom config. If you need help, I can show you

**Reply by Enigama** [reply]
> Yeah, seems easy to do. You can already add this to a custom config. If you need help, I can show you

@williambdean yes, please

**Reply by williambdean** [reply]
See here #1009
